src/DataDeps.jl

```diff
@@ -26,8 +26,12 @@ struct DataDep{H, R, F, P} <: AbstractDataDep
     extra_message::String
 end
 
-macro datadep_str(name)
-    :(resolve(registry[$(esc(name))], @__FILE__))
+macro datadep_str(path)
+    parts = splitpath(path)
```

Comment:
random thought: does `splitpath` recognize `"/"` as a folder separator in windows as well?

Reply:
I think so.
CI will tell me if it doesn't.